tutorials/manage-instances-with-serverless-framework/index.mdx

```diff
@@ -0,0 +1,147 @@
+---
+meta:
+  title: Managing compute Instances with the serverless framework
```

Comment:
Could the title just be "Managing Instances with the serverless framework"? or "Scaleway Instances"?

Reply:
```suggestion
  title: Managing Scaleway Instances with the serverless framework
```